CIエラー: TestAccSakuraCloudDisk_with_Server

https://github.com/sacloud/terraform-provider-sakuracloud/actions/runs/12040846996

`make testfake`でのみ発生する模様

```
=== CONT  TestAccSakuraCloudDisk_with_Server
    resource_sakuracloud_disk_test.go:91: Step 1/2 error: Check failed: Check 21/21 error: sakuracloud_server.foobar: Attribute 'ip_address' expected to be set
--- FAIL: TestAccSakuraCloudDisk_with_Server (5.49s)
```

--- a/sakuracloud/resource_sakuracloud_disk_test.go
+++ b/sakuracloud/resource_sakuracloud_disk_test.go
@@ -82,6 +82,8 @@ func TestAccSakuraCloudDisk_basic(t *testing.T) {
 }
 
 func TestAccSakuraCloudDisk_with_Server(t *testing.T) {
+	skipIfFakeModeEnabled(t) // FakeModeだとip_address指定が動かないためスキップする
+
 	diskResourceName := "sakuracloud_disk.foobar"
 	serverResourceName := "sakuracloud_server.foobar"
 	rand := randomName()

